config/routes.rb

```diff
@@ -1,4 +1,12 @@
 Rails.application.routes.draw do
+  resources :outcomes
+  resources :complaints
+  resources :ipeds_ic_pies
+  resources :ipeds_ic_ays
+  resources :ipeds_hds
+  resources :ipeds_ics
+  resources :settlements
+  resources :hcms
```

Comment:
Are these routes actually exposed? Looking at the corresponding controller stubs above, lets not add / expose these routes until the controller logic is actually implemented.